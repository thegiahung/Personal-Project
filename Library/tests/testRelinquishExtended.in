add member Python
add member Java
add member Ruby
add member Tom Cruise
add collection books.csv
rent 100000 111111
relinquish 100000 111111
rent 100001 111111
rent 100002 111111
relinquish 100002 111111
relinquish 100002 200000
rent 100003 111115
rent 100003 111116
rent 100003 111117
relinquish 100003 111115
relinquish all 100003
exit
