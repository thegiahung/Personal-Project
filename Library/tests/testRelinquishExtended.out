user: Success.

user: Success.

user: Success.

user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Book is currently unavailable.

user: Unable to return book.

user: No such book in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Ending Library process.
